### PR TITLE
Adapting to new scheduling protocol

### LINKED ISSUE #131: Adapt to changed scheduler protocol
With the new scheduler, `CompletionMessage` now longer returns a list of new ticks, but only a single tick that the external simulation likes to be triggered with: https://github.com/ie3-institute/simona/pull/592

### PR DESCRIPTION
**Pull request by sebastian-peter**
Resolves #131 (also gives an explanation)

### CODE REVIEW

File: src/main/java/edu/ie3/simona/api/simulation/ExtSimulation.java [hunk 1]
```diff
@@ -79,15 +80,15 @@ private boolean takeAndHandleMessage() throws InterruptedException {
    *
    * @return a list of future ticks at which this external simulation wants to be triggered.
```
Comment (danielfeismann) [inline src/main/java/edu/ie3/simona/api/simulation/ExtSimulation.java:81]:
```suggestion
   * @returns an Optional of the next future tick at which this external simulation wants to be triggered.
```

**Review by danielfeismann — APPROVED** [review]
Looks good to me, thanks! Just one very minor point.

File: src/main/java/edu/ie3/simona/api/simulation/ExtSimulation.java (hunk 1 above)
Comment (sebastian-peter) [reply src/main/java/edu/ie3/simona/api/simulation/ExtSimulation.java:81]: Good catch. Also adapted JavaDoc further below.

**Review by danielfeismann — APPROVED** [review]
LGTM, thanks for the work done here!

**Pull request merged**